model/licensing.py

```diff
@@ -800,7 +800,13 @@ def loan_to(self, patron_or_client, start=None, end=None, fulfillment=None, exte
         if isinstance(patron_or_client, Patron):
             loan, is_new = get_one_or_create(
                 _db, Loan, patron=patron_or_client, license_pool=self,
-                create_method_kwargs=kwargs)
+                create_method_kwargs=kwargs
+            )
+
+            # This action creates uncertainty about what the patron's
+            # loan activity actually is. We'll need to sync with the
+            # vendor APIs.
+            patron_or_client.last_loan_activity_sync = None
```

Comment:
The loan_to method didn't have a unit test of its own so I added a basic one while testing this.